tests/test-wp-parsely.php

```diff
@@ -250,7 +250,19 @@ function test_http_canonicals()
         $ppage = self::$parsely->insert_parsely_page();
         $this->assertTrue(strpos($ppage['url'], 'https', 0) == 0);
     }
-
+    
+    
+      
+    function test_amp_analytics() {  
+      	$options = get_option('parsely');
+        $post_array = $this->create_test_post_array();
+        $post = $this->factory->post->create($post_array);
+        $this->go_to('/?p=' . $post);
+        $ppage = self::$parsely->insert_parsely_amp_tracking();
+        $this->assertTrue(strpos($ppage['url'], '/amp') == 0);
+        $this->assertFalse(strpos($ppage['url'], '/amp'));
+        $this->assertTrue(strpos($ppage['url'], 'amp=1') == 0);
+        $this->assertFalse(strpos($ppage['url'], 'amp=1'));       
```

Comment:
We'll then remove these four assertions and replace them with the following two: 

```
$this->assertTrue(strpos($output, 'amp-analytics id="wp-parsely"') > 0);
$this->assertTrue(strpos($output, '{"vars":{"apikey":"blog.parsely.com"}}') > 0);
```

`blog.parsely.com` is the value given for `apikey` in the test stub [here](https://github.com/Parsely/wp-parsely/blob/master/tests/test-wp-parsely.php#L64)